project/src/main/java/com/google/sps/data/PlacesFetcher.java

```diff
@@ -25,27 +26,38 @@
 import com.google.maps.errors.ApiException;
 import com.google.maps.model.PlacesSearchResult;
 import com.google.maps.PlacesApi;
+import com.google.gson.Gson;
+import com.google.gson.reflect.TypeToken;
+import com.google.gson.stream.JsonReader;
 import java.io.IOException;
+import java.io.InputStreamReader;
+import java.lang.reflect.Type;
 import java.util.ArrayList;
 import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
 
 public class PlacesFetcher {
 
-    /** The type of places that will be searched is RESTAURANT. Since most places
-     * that deliver food are not tagged as "MEAL-DELIVERY" type at Google Places but
-     * rather as "RESTAURANT" this is the most suitable type to search for.
-     */
+    // The type of places that will be searched is RESTAURANT. Since most places
+    // that deliver food are not tagged as "MEAL-DELIVERY" type at Google Places but
+    // rather as "RESTAURANT" this is the most suitable type to search for.
     private static final PlaceType TYPE = PlaceType.RESTAURANT;
 
-    /** In this radius around "LOCATION" places will be searched. */
+    // In this radius around "LOCATION" places will be searched.
     private static final int SEARCH_RADIUS = 5000;
-    // TODO(M1): check at least 10 results, and if less extend radius
 
-    /** The entry point for a Google GEO API request. */
+    // The entry point for a Google GEO API request.
     private static final GeoApiContext CONTEXT = new GeoApiContext.Builder()
         .apiKey(System.getenv("API_KEY"))
         .build();
 
+    // The path of the configuration file containing the mapping of cuisines to search words.
+    private static final String CUISINES_SEARCH_WORDS_CONFIG_PATH  = "cuisinesSearchWords.json";
+
+    // A mapping between cuisines and text search words. */
+    private static final Map<String, List<String>> CUISINE_TO_SEARCH_WORDS = getCuisinesMap();
```

Comment:
Should change here to ImmutableMap as well.

Reply:
Sure, I changed it